### DISCUSSION TITLE
vue2-gtm support for GA4.

### DISCUSSION THREAD

**Discussion by cszczotka** [gtm-support/vue-gtm#355] [Q&A]
Hi,
First question does @gtm-support/vue2-gtm version 1.3.0 supports GA4 ? 

I have vue.js 2.6.x application where is used deprecated npm https://www.npmjs.com/package/vue-gtm version 2.0.0.  this works fine with GA3 but for GA4 we don't see some reports. I'm trying to migrate to  @gtm-support/vue2-gtm 1.3.0 which I can compile and run with my application. For @gtm-support/vue2-gtm 2.0.0 I'm getting such compilation error:

ERROR in ./node_modules/@gtm-support/vue2-gtm/dist/index.js 58:44
Module parse failed: Unexpected token (58:44)
You may need an appropriate loader to handle this file type, currently no loaders are configured to process this file. See https://webpack.js.org/concepts#loaders
|       ...(_a = to.meta) == null ? void 0 : _a.gtmAdditionalEventData
|     };
>     const baseUrl = vueRouter.options.base ?? "";
|     let fullUrl = baseUrl;
|     if (!fullUrl.endsWith("/")) {
 @ ./src/modules/google-tag-manager/index.ts 3:0-43 23:20-26
 @ ./src/modules/client.ts
 @ ./core/app.ts
 @ ./core/client-entry.ts
 @ multi webpack-hot-middleware/client @babel/polyfill ./core/client-entry.ts

Second question do you know what can be reason this compilation error ?
Thanks

**Comment by Shinigami92** [comment]
> [...] supports GA4

no, this is only for GTM, not for GA
